osm2streets/src/lib.rs

```diff
@@ -545,14 +547,7 @@ impl Road {
         // If there's a sidewalk on only one side, adjust the true center of the road.
         // TODO I don't remember the rationale for doing this in the first place. What if there's a
```

Comment:
... I still don't understand / remember what this entire method is trying to do. I suspect we'll have to finally figure that out when implementing placement tags

Reply:
Yep, this is exactly the complexity that the placement tag addresses, thereby defining what "center" means. From memory it means the center of the road surface, but I'll dig into that.